pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminSupplier.java

```diff
@@ -103,6 +103,9 @@ private static void applyRootParamsToAdminBuilder(PulsarAdminBuilder adminBuilde
         if (isNotBlank(rootParams.tlsProvider)) {
             adminBuilder.sslProvider(rootParams.tlsProvider);
         }
+        if (isNotBlank(rootParams.tlsTrustCertsFilePath)) {
+            adminBuilder.tlsTrustCertsFilePath(rootParams.tlsTrustCertsFilePath);
```

Comment:
Quickly reading the source code, it looks like we read it from the `client.conf` file here:

https://github.com/apache/pulsar/blob/fb0cb1ce71a14b8109c828a8d9fcfc15b84e6a09/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java#L123

I mention it in case anyone needs a workaround.

Reply:
@michaeljmarshall there are two ways to configure variables: configure in conf file or pass as arguments in CLI. right now, it only reads from `conf/client.conf` file but it ignores if user provide as a cli argument that's because it doesn't check `rootParams.tlsTrustCertsFilePath` which is parsed as CLI args.